app/assets/stylesheets/application.css.scss

```diff
@@ -246,6 +259,16 @@ main {
   display: block;
   box-sizing: initial;
 }
+
+.headers{
+  padding-bottom: 0px;
```

Comment:
Properties should be ordered margin-bottom, margin-top, padding-bottom<br>`0px` should be written without units as `0`